notebooks/official/migration/sdk-custom-xgboost-prebuilt-container.ipynb

```diff
@@ -518,13 +446,13 @@
         "  - \\_\\_init\\_\\_.py\n",
         "  - task.py\n",
         "\n",
-        "The files `setup.cfg` and `setup.py` are the instructions for installing the package into the operating environment of the Docker image.\n",
+        "The files *setup.cfg* and *setup.py* are the instructions for installing the package into the operating environment of the Docker image.\n",
         "\n",
-        "The file `trainer/task.py` is the Python script for executing the custom training job. *Note*, when we referred to it in the worker pool specification, we replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
+        "The file *trainer/task.py* is the Python script for executing the custom training job. *Note*, when trainer/task.py is referred to in the worker pool specification, the directory slash is replaced with a dot and the file suffix (.py) is dropped (*trainer.task*).\n",
```

Comment:
- change "Note" to: "*Note*: When `trainer/task.py` is referred to in the worker pool specification, the directory slash is replaced with a dot and the file suffix (`.py`) is dropped (`trainer.task`)."